### DISCUSSION TITLE
How to change the default bg pic from a theme?

### DISCUSSION THREAD

**Discussion by SmartPhoneLover** [Lissy93/dashy#655] [Help and Support] [answered]
Hello, I'm currently using the 'Adventure' theme, which has a background image configured by default (purple, from dark to light).
So, can I change that background image by another image of my choise, or maybe a solid color?

Thank you!

**Comment by lissy93** [comment, marked as answer]
Hi @SmartPhoneLover,
Apologies, I totally forgot to reply to this.
In the latest PR (#663, specifically 3417b46eeec3cf96475cea54f725099c6f204bd6) I added an additional theme named `adventure-basic` for you. If you select that theme, it won't load the original image asset, and you can then go ahead and set your own background image.
Hope that helps :)

**Reply by SmartPhoneLover** [reply]
Thank you very much!!!

**Reply by lissy93** [reply]
Anytime!
